apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -48,16 +48,13 @@ export default class AnimationPickerBody extends React.Component {
 
   componentDidMount() {
     this.scrollListContainer = React.createRef();
-  }
-
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    if (this.props.defaultQuery !== nextProps.defaultQuery) {
+    if (this.props.defaultQuery) {
```

Comment:
does moving this logic to `componentDidMount` cause any behavior change?

Reply:
Not that I could find. This particular change I worked through with @madelynkasula's help. I remember we basically had a try-and-see mentality around the move.

Reply:
i think the button in the backgrounds block (used to say "More," now says "Backgrounds") is the only place that uses the `defaultQuery` prop, so as long as we don't regress there we should be good